WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/CouponExpiryDateView.swift

```diff
@@ -16,31 +18,59 @@ struct CouponExpiryDateView: View {
                     DatePicker("Date picker", selection: $date, displayedComponents: .date)
                         .environment(\.timeZone, timezone)
                         .datePickerStyle(GraphicalDatePickerStyle())
-                        .onChange(of: date) { newDate in
-                            completion(newDate)
-                        }
+                        .onChange(of: date, perform: { newDate in
+                                isRemovalEnabled = true
+                            })
```

Comment:
Super nit: we can use trailing closure to make this neater, and omit `newDate` since we're not using it.
```suggestion
                        .onChange(of: date) { _ in
                            isRemovalEnabled = true
                        }
```

Reply:
Nice! I ended up removing the trailing closure declaration that was before without thinking, it's fixed now: https://github.com/woocommerce/woocommerce-ios/pull/6953/commits/3954649a72b3ba26ef05221fc3959f171e12fe72